minor changes (#29)
yaq-project/yaq-protocol-publication: paper.tex

The Wright Group uses simulated hardware to enable client development away from the active laboratory computers.
This allows Wright Group researchers to create polished client interfaces without interrupting  ongoing experiments.
Clients developed by the Wright Group have proved flexible enough to be used on four laser systems, each with different complements of hardware.

> The Wright Group expects to spend less energy developing control software and more energy developing creative spectroscopy experiments in future work.

Reviewer: Moving forward, the Wright Group will spend less energy developing control software and more energy developing creative spectroscopy experiments.
```suggestion
Moving forward, the Wright Group will spend less energy developing control software and more energy developing creative spectroscopy experiments.
```
